core/movieinfo.py

```diff
@@ -221,6 +221,49 @@ def get_imdbid(self, tmdbid=None, title=None, year=''):
             logging.error('Error attempting to get IMDBID from TMDB.', exc_info=True)
             return ''
 
+    def get_suggestions(self, section, tmdbid):
+        ''' get popular movies from TMDB
+
+        Returns list of results
+        '''
+
+        logging.info('Grabbing popular movies from  TheMovieDB')
+
+        if section == 'toprated':
+            url = 'https://api.themoviedb.org/3/movie/top_rated?language=en-US&include_adult=false&region=US'
+        elif section == 'popular':
+            url = 'https://api.themoviedb.org/3/movie/popular?language=en-US&include_adult=false&region=US'
+        elif section == 'nowplaying':
+            url = 'https://api.themoviedb.org/3/movie/now_playing?language=en-US&include_adult=false&region=US'
+        elif section == 'upcoming':
+            url = 'https://api.themoviedb.org/3/movie/upcoming?language=en-US&include_adult=false&region=US'
+        elif section == 'recommendation':
+            rmovie = core.sql.get_random_movie()
+            url = 'https://api.themoviedb.org/3/movie/' + str( rmovie.get('tmdbid')) + '/recommendations?language=en-US&include_adult=false'
+        elif section == 'similar':
+            url = 'https://api.themoviedb.org/3/movie/' + str(tmdbid) + '/recommendations?language=en-US&include_adult=false'
+
```

Comment:
This wont affect usage, but is more of a stylistic choice.

The only conditions you need to check is for are `similar` and `recommendation` since they require a different url format. Otherwise you can construct `url` with `'https://api.themoviedb.org/3/movie/{}?language=en-US&include_adult=false&region=US'.format(section)`. This makes it easier to read and troubleshoot by not repeating the base url more than necessary.